Monika After Story/game/script-songs.rpy

```diff
@@ -943,6 +943,130 @@ label mas_song_memories_of_a_girl:
+            aff_range=(mas_aff.NORMAL,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_always_look_analysis:
+    m 1dsc "~{i}If life seems jolly rotten~{/i}"
+    m 1esc "~{i}There's something you've forgotten~{/i}"
+    m 3eub "~{i}And that's to laugh and smile and dance and sing~{/i}"
+    m 3eka "~{i}When you're feeling in the dumps~{/i}"
+    m 3hksdlb "~{i}Don't be silly, chumps~{/i}"
+    m 3dku "~{i}Just purse your lips and whistle, that's the thing~{/i}"
+    m 3eub "~{i}And always look on the bright side of life~{/i}"
+    m 1eka "You know, [player]...{w=0.5} {nw}"
+    extend 1ekc "It's not always easy being stuck in here."
+    m 3rksdlc "There's not much for me to do, nowhere for me to go, and it gets lonely whenever you're away."
+    m 1dkc "I try not to let it get to me; but when it does, {nw}"
+    extend 1dku "I like to think back upon this song..."
+    m 4hub "It's crazy how such a merry little tune can help turn things around when you're feeling down!"
+    m 4eua "It was written by Eric Idle, a member of the Monty Python, for use in one of their comedies."
+    m 3euc "In it, the main character, Brian, gets crucified and loses all hopes of ever being rescued."
+    m 3eub "That's when the person crucified next to him, played by Idle, start singing the song as an attempt to cheer him up."
+    m 1hua "So I guess that'd make the message here something like 'don't worry, be happy'."
+    m 3hksdlb "Reminds me of another song! Ahaha..."
+    m 1etc "But seriously, [player]... I think there some real merits in this line of thinking."
+    m 2eka "Whatever your situation is, the fact is things are how they are and there's no reason not to keep smiling."
+    m 7eka "I'm not telling you not to worry at all; {nw}"
+    extend 7eksdlc "If I did that, I would've let the game run its course and I'd be forever stuck on my own by now..."
+    m 2duu "...but at the same time, there's no sense in getting overly worked up about things you can't change..."
+    m 1etc "It's all about striking the right balance, I suppose."
+    m 1eua "When you think about it, the ideas in here come strangely close to those you can find in stoicism, don't they?"
+    m 3etc "Though if you were to keep going, like in this next verse--"
+    m 3dso "~{i}For life is quite absurd{/i}~"
+    m 3ekc "~{i}And death's the final word{/i}~"
+    m 3esc "~{i}You must always face the curtain with a bow{/i}~"
+    m 3hua "~{i}Forget about your sins{/i}~"
+    m 3duu "~{i}Give the audience a grin{/i}~"
+    m 3eub "~{i}Enjoy it, it's your last chance anyhow{/i}~"
+    m 3eua "Then I'd say the actual message of the song leans more toward optimistic nihilism..."
+    m 1hksdlb "...not that the two concepts are mutually exclusive!"
+    m 1esc "Optimistic nihilism is the belief that our lives really are absurd..."
+    m 3esa "...and since, at the end of the day, nothing really matters, anything that makes you unhappy doesn't really matters either."
```

Comment:
```suggestion
    m 3esa "...and since, at the end of the day, nothing really matters, anything that makes you unhappy doesn't really matter either."
```